package.json

```diff
@@ -21,7 +21,7 @@
     "lint:js": "eslint --fix --cache lib test",
     "lint:json": "prettier --write codecov.yml .circleci/config.yml .eslintrc",
     "lint:package": "prettier --write package.json --plugin=prettier-plugin-package",
-    "posttest": "node set-webpack-version.js \"^4.44.2\" && npm install",
+    "posttest": "node set-webpack-version.js \">=4.44.2\" && npm install",
```

Comment:
please revert this change

Reply:
Should this become '^4.44.2 || ^5.46.0' just like the other setting? Because if I simply revert it, testing will alter package.json and perhaps cause committing unwanted changes.

Reply:
There's a larger fix here that's needed related to the set-webpack-version script to prevent bad overwrites. I'll do that after your PR.

Reply:
Cool